<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="40" passed="27" failed="13" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-03T09:12:59 AEST" name="PetStoreSuite" finished-at="2024-08-03T09:14:28 AEST" duration-ms="88516">
    <groups>
    </groups>
    <test started-at="2024-08-03T09:12:59 AEST" name="PetStoreTests" finished-at="2024-08-03T09:14:28 AEST" duration-ms="88516">
      <class name="com.au.petstore.testsAPI.GetPetByStatus">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:12:59 AEST" name="generateRandomID" finished-at="2024-08-03T09:12:59 AEST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="getPetByInvalidStatus()[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:13:22 AEST" name="getPetByInvalidStatus" finished-at="2024-08-03T09:13:24 AEST" duration-ms="1434" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.GetPetByStatus.getPetByInvalidStatus(GetPetByStatus.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByInvalidStatus -->
        <test-method signature="getPetByRemovingStatus()[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:13:24 AEST" name="getPetByRemovingStatus" finished-at="2024-08-03T09:13:25 AEST" duration-ms="1206" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <405>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <405>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.GetPetByStatus.getPetByRemovingStatus(GetPetByStatus.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByRemovingStatus -->
        <test-method signature="getPetByStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:13:25 AEST" name="getPetByStatus" data-provider="status" finished-at="2024-08-03T09:13:28 AEST" duration-ms="3384" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[available]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByStatus -->
        <test-method signature="getPetByStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:13:28 AEST" name="getPetByStatus" data-provider="status" finished-at="2024-08-03T09:13:32 AEST" duration-ms="3381" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[sold]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByStatus -->
        <test-method signature="getPetByStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.GetPetByStatus@1abc9f14]" started-at="2024-08-03T09:13:32 AEST" name="getPetByStatus" data-provider="status" finished-at="2024-08-03T09:13:35 AEST" duration-ms="2947" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pending]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByStatus -->
      </class> <!-- com.au.petstore.testsAPI.GetPetByStatus -->
      <class name="com.au.petstore.testsAPI.DeletePet">
        <test-method is-config="true" signature="cleanAllureResultsFolder()[pri:0, instance:com.au.petstore.testsAPI.DeletePet@1f966492]" started-at="2024-08-03T09:12:59 AEST" name="cleanAllureResultsFolder" finished-at="2024-08-03T09:12:59 AEST" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanAllureResultsFolder -->
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.DeletePet@1f966492]" started-at="2024-08-03T09:12:59 AEST" name="generateRandomID" finished-at="2024-08-03T09:12:59 AEST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="deleteNonExistencePetIDFromTheStore()[pri:0, instance:com.au.petstore.testsAPI.DeletePet@1f966492]" started-at="2024-08-03T09:12:59 AEST" name="deleteNonExistencePetIDFromTheStore" finished-at="2024-08-03T09:13:10 AEST" duration-ms="10907" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteNonExistencePetIDFromTheStore -->
        <test-method signature="deletePetFromTheStoreWithValidPetID()[pri:0, instance:com.au.petstore.testsAPI.DeletePet@1f966492]" started-at="2024-08-03T09:13:11 AEST" name="deletePetFromTheStoreWithValidPetID" finished-at="2024-08-03T09:13:14 AEST" duration-ms="2975" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePetFromTheStoreWithValidPetID -->
        <test-method signature="deletePetIDFromTheStoreWithoutMandatoryFieldPetID()[pri:0, instance:com.au.petstore.testsAPI.DeletePet@1f966492]" started-at="2024-08-03T09:13:14 AEST" name="deletePetIDFromTheStoreWithoutMandatoryFieldPetID" finished-at="2024-08-03T09:13:15 AEST" duration-ms="1407" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePetIDFromTheStoreWithoutMandatoryFieldPetID -->
      </class> <!-- com.au.petstore.testsAPI.DeletePet -->
      <class name="com.au.petstore.testsAPI.GetPetByID">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.GetPetByID@6e57b5e9]" started-at="2024-08-03T09:13:15 AEST" name="generateRandomID" finished-at="2024-08-03T09:13:17 AEST" duration-ms="1529" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="getPetByID()[pri:0, instance:com.au.petstore.testsAPI.GetPetByID@6e57b5e9]" started-at="2024-08-03T09:13:17 AEST" name="getPetByID" finished-at="2024-08-03T09:13:18 AEST" duration-ms="1456" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByID -->
        <test-method signature="getDeletePetAsNotFound()[pri:0, instance:com.au.petstore.testsAPI.GetPetByID@6e57b5e9]" started-at="2024-08-03T09:13:18 AEST" name="getDeletePetAsNotFound" finished-at="2024-08-03T09:13:21 AEST" depends-on-methods="com.au.petstore.testsAPI.GetPetByID.getPetByID" duration-ms="2838" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDeletePetAsNotFound -->
        <test-method signature="getPetByInvalidID()[pri:0, instance:com.au.petstore.testsAPI.GetPetByID@6e57b5e9]" started-at="2024-08-03T09:13:21 AEST" name="getPetByInvalidID" finished-at="2024-08-03T09:13:22 AEST" duration-ms="1534" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPetByInvalidID -->
      </class> <!-- com.au.petstore.testsAPI.GetPetByID -->
      <class name="com.au.petstore.testsAPI.PostUploadPetImage">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.PostUploadPetImage@77cb9cd1]" started-at="2024-08-03T09:14:00 AEST" name="generateRandomID" finished-at="2024-08-03T09:14:02 AEST" duration-ms="1460" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="updateExistingPetWithImage()[pri:0, instance:com.au.petstore.testsAPI.PostUploadPetImage@77cb9cd1]" started-at="2024-08-03T09:14:02 AEST" name="updateExistingPetWithImage" finished-at="2024-08-03T09:14:03 AEST" duration-ms="1579" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExistingPetWithImage -->
        <test-method signature="updatePetWithDeletedPetID()[pri:0, instance:com.au.petstore.testsAPI.PostUploadPetImage@77cb9cd1]" started-at="2024-08-03T09:14:03 AEST" name="updatePetWithDeletedPetID" finished-at="2024-08-03T09:14:06 AEST" duration-ms="2953" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <404>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <404>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostUploadPetImage.updatePetWithDeletedPetID(PostUploadPetImage.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithDeletedPetID -->
        <test-method signature="updatePetWithMandatoryFields()[pri:0, instance:com.au.petstore.testsAPI.PostUploadPetImage@77cb9cd1]" started-at="2024-08-03T09:14:06 AEST" name="updatePetWithMandatoryFields" finished-at="2024-08-03T09:14:08 AEST" duration-ms="1329" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <200>
     but: was <404>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <200>
     but: was <404>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostUploadPetImage.updatePetWithMandatoryFields(PostUploadPetImage.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithMandatoryFields -->
        <test-method signature="updatePetWithoutMandatoryFields()[pri:0, instance:com.au.petstore.testsAPI.PostUploadPetImage@77cb9cd1]" started-at="2024-08-03T09:14:08 AEST" name="updatePetWithoutMandatoryFields" finished-at="2024-08-03T09:14:09 AEST" duration-ms="1499" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithoutMandatoryFields -->
      </class> <!-- com.au.petstore.testsAPI.PostUploadPetImage -->
      <class name="com.au.petstore.testsAPI.PutUpdateAnExistingPet">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:09 AEST" name="generateRandomID" finished-at="2024-08-03T09:14:11 AEST" duration-ms="1533" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="notAbleToCreatePetAsEmptyObjectError400()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:11 AEST" name="notAbleToCreatePetAsEmptyObjectError400" finished-at="2024-08-03T09:14:12 AEST" duration-ms="1431" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PutUpdateAnExistingPet.notAbleToCreatePetAsEmptyObjectError400(PutUpdateAnExistingPet.java:122)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetAsEmptyObjectError400 -->
        <test-method signature="notAbleToCreatePetAsIDIsMissingError500()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:12 AEST" name="notAbleToCreatePetAsIDIsMissingError500" finished-at="2024-08-03T09:14:13 AEST" duration-ms="1279" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetAsIDIsMissingError500 -->
        <test-method signature="notAbleToCreatePetWithoutBodyError405()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:13 AEST" name="notAbleToCreatePetWithoutBodyError405" finished-at="2024-08-03T09:14:15 AEST" duration-ms="1508" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetWithoutBodyError405 -->
        <test-method signature="notAbleToUpdatePetBadRequestError400()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:15 AEST" name="notAbleToUpdatePetBadRequestError400" finished-at="2024-08-03T09:14:16 AEST" duration-ms="1501" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PutUpdateAnExistingPet.notAbleToUpdatePetBadRequestError400(PutUpdateAnExistingPet.java:107)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToUpdatePetBadRequestError400 -->
        <test-method signature="updatePetWithDeletedPetID()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:16 AEST" name="updatePetWithDeletedPetID" finished-at="2024-08-03T09:14:19 AEST" duration-ms="2863" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <404>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <404>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PutUpdateAnExistingPet.updatePetWithDeletedPetID(PutUpdateAnExistingPet.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithDeletedPetID -->
        <test-method signature="updatePetWithDifferentStatusForSamePetID(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:19 AEST" name="updatePetWithDifferentStatusForSamePetID" data-provider="status" finished-at="2024-08-03T09:14:21 AEST" duration-ms="1536" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[available]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithDifferentStatusForSamePetID -->
        <test-method signature="updatePetWithDifferentStatusForSamePetID(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:21 AEST" name="updatePetWithDifferentStatusForSamePetID" data-provider="status" finished-at="2024-08-03T09:14:22 AEST" duration-ms="1328" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[sold]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithDifferentStatusForSamePetID -->
        <test-method signature="updatePetWithDifferentStatusForSamePetID(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:22 AEST" name="updatePetWithDifferentStatusForSamePetID" data-provider="status" finished-at="2024-08-03T09:14:24 AEST" duration-ms="1434" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pending]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithDifferentStatusForSamePetID -->
        <test-method signature="updatePetWithMandatoryFields()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:24 AEST" name="updatePetWithMandatoryFields" finished-at="2024-08-03T09:14:25 AEST" duration-ms="1248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithMandatoryFields -->
        <test-method signature="updatePetWithoutMandatoryFieldsError400()[pri:0, instance:com.au.petstore.testsAPI.PutUpdateAnExistingPet@971e903]" started-at="2024-08-03T09:14:25 AEST" name="updatePetWithoutMandatoryFieldsError400" finished-at="2024-08-03T09:14:26 AEST" duration-ms="1506" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PutUpdateAnExistingPet.updatePetWithoutMandatoryFieldsError400(PutUpdateAnExistingPet.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithoutMandatoryFieldsError400 -->
      </class> <!-- com.au.petstore.testsAPI.PutUpdateAnExistingPet -->
      <class name="com.au.petstore.testsAPI.PostPetInStoreWithFormData">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.PostPetInStoreWithFormData@24e08d59]" started-at="2024-08-03T09:13:53 AEST" name="generateRandomID" finished-at="2024-08-03T09:13:55 AEST" duration-ms="1289" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="updateExistingPetWithNameAndStatus(java.lang.String,java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostPetInStoreWithFormData@24e08d59]" started-at="2024-08-03T09:13:55 AEST" name="updateExistingPetWithNameAndStatus" data-provider="nameAndStatus" finished-at="2024-08-03T09:13:56 AEST" duration-ms="1541" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Jack]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[available]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExistingPetWithNameAndStatus -->
        <test-method signature="updateExistingPetWithNameAndStatus(java.lang.String,java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostPetInStoreWithFormData@24e08d59]" started-at="2024-08-03T09:13:56 AEST" name="updateExistingPetWithNameAndStatus" data-provider="nameAndStatus" finished-at="2024-08-03T09:13:57 AEST" duration-ms="1360" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Jo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sold]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExistingPetWithNameAndStatus -->
        <test-method signature="updateExistingPetWithNameAndStatus(java.lang.String,java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostPetInStoreWithFormData@24e08d59]" started-at="2024-08-03T09:13:57 AEST" name="updateExistingPetWithNameAndStatus" data-provider="nameAndStatus" finished-at="2024-08-03T09:13:59 AEST" duration-ms="1398" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Mike]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pending]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExistingPetWithNameAndStatus -->
        <test-method signature="updatePetWithInvalidFields()[pri:0, instance:com.au.petstore.testsAPI.PostPetInStoreWithFormData@24e08d59]" started-at="2024-08-03T09:13:59 AEST" name="updatePetWithInvalidFields" finished-at="2024-08-03T09:14:00 AEST" duration-ms="1398" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <200>
     but: was <415>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <200>
     but: was <415>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostPetInStoreWithFormData.updatePetWithInvalidFields(PostPetInStoreWithFormData.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetWithInvalidFields -->
      </class> <!-- com.au.petstore.testsAPI.PostPetInStoreWithFormData -->
      <class name="com.au.petstore.testsAPI.PostCreatePetAPI">
        <test-method is-config="true" signature="generateRandomID()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:12:59 AEST" name="generateRandomID" finished-at="2024-08-03T09:12:59 AEST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateRandomID -->
        <test-method signature="createPetWIthSameID()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:35 AEST" name="createPetWIthSameID" finished-at="2024-08-03T09:13:38 AEST" duration-ms="2878" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <409>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <409>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostCreatePetAPI.createPetWIthSameID(PostCreatePetAPI.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPetWIthSameID -->
        <test-method signature="createPetWithMandatoryFields()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:38 AEST" name="createPetWithMandatoryFields" finished-at="2024-08-03T09:13:39 AEST" duration-ms="1523" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPetWithMandatoryFields -->
        <test-method signature="notAbleToCreatePetAsEmptyObjectError400()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:39 AEST" name="notAbleToCreatePetAsEmptyObjectError400" finished-at="2024-08-03T09:13:41 AEST" duration-ms="1521" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostCreatePetAPI.notAbleToCreatePetAsEmptyObjectError400(PostCreatePetAPI.java:142)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetAsEmptyObjectError400 -->
        <test-method signature="notAbleToCreatePetAsIDIsMissingError500()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:41 AEST" name="notAbleToCreatePetAsIDIsMissingError500" finished-at="2024-08-03T09:13:43 AEST" duration-ms="2672" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetAsIDIsMissingError500 -->
        <test-method signature="notAbleToCreatePetAsResourceIsMissingError404()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:43 AEST" name="notAbleToCreatePetAsResourceIsMissingError404" finished-at="2024-08-03T09:13:45 AEST" duration-ms="1316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetAsResourceIsMissingError404 -->
        <test-method signature="notAbleToCreatePetBadRequestError400()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:45 AEST" name="notAbleToCreatePetBadRequestError400" finished-at="2024-08-03T09:13:46 AEST" duration-ms="1426" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostCreatePetAPI.notAbleToCreatePetBadRequestError400(PostCreatePetAPI.java:114)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetBadRequestError400 -->
        <test-method signature="notAbleToCreatePetWithoutBodyError405()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:46 AEST" name="notAbleToCreatePetWithoutBodyError405" finished-at="2024-08-03T09:13:48 AEST" duration-ms="1450" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notAbleToCreatePetWithoutBodyError405 -->
        <test-method signature="postAPIToCreatePetWithDifferentStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:48 AEST" name="postAPIToCreatePetWithDifferentStatus" data-provider="status" finished-at="2024-08-03T09:13:49 AEST" duration-ms="1314" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[available]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postAPIToCreatePetWithDifferentStatus -->
        <test-method signature="postAPIToCreatePetWithDifferentStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:49 AEST" name="postAPIToCreatePetWithDifferentStatus" data-provider="status" finished-at="2024-08-03T09:13:50 AEST" duration-ms="1529" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[sold]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postAPIToCreatePetWithDifferentStatus -->
        <test-method signature="postAPIToCreatePetWithDifferentStatus(java.lang.String)[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:50 AEST" name="postAPIToCreatePetWithDifferentStatus" data-provider="status" finished-at="2024-08-03T09:13:52 AEST" duration-ms="1431" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pending]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postAPIToCreatePetWithDifferentStatus -->
        <test-method signature="postAPIToCreatePetWithoutMandatoryFieldsError400()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:13:52 AEST" name="postAPIToCreatePetWithoutMandatoryFieldsError400" finished-at="2024-08-03T09:13:53 AEST" duration-ms="1462" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: is <400>
     but: was <200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: is <400>
     but: was <200>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.au.petstore.testsAPI.PostCreatePetAPI.postAPIToCreatePetWithoutMandatoryFieldsError400(PostCreatePetAPI.java:99)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postAPIToCreatePetWithoutMandatoryFieldsError400 -->
        <test-method is-config="true" signature="deleteRandomPetID()[pri:0, instance:com.au.petstore.testsAPI.PostCreatePetAPI@6a0ac48e]" started-at="2024-08-03T09:14:26 AEST" name="deleteRandomPetID" finished-at="2024-08-03T09:14:28 AEST" duration-ms="1432" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRandomPetID -->
      </class> <!-- com.au.petstore.testsAPI.PostCreatePetAPI -->
    </test> <!-- PetStoreTests -->
  </suite> <!-- PetStoreSuite -->
</testng-results>
